docs/userman/advanced_features/auth/authorization.md

```diff
@@ -305,3 +305,26 @@ with Authorize(SystemCredentials()):
     # from the SystemCredentials
     ...
 ```
+
+# Run the Orchestrator service with specific Credentials
+
+Running the Orchestrator service requires authorization with the `TAIPY_EXECUTOR` role.
```

Comment:
I believe the role should be editor.

```suggestion
Running the Orchestrator service requires authorization with the `TAIPY_EDITOR` role.
```

NOte that in the future, we will need to update this replacing predefined roles by appropriate rights.

Reply:
You're right. My bad